package.json

```diff
@@ -64,12 +65,16 @@
     "bluebird": "3.5.1",
     "chalk": "2.4.2",
     "clear-require": "1.0.1",
+    "cli-progress": "3.12.0",
     "debug": "2.6.9",
     "devtools": "8.39.0",
+    "edgedriver": "5.6.1",
     "error-stack-parser": "2.1.4",
     "expect-webdriverio": "3.6.0",
+    "extract-zip": "2.0.1",
```

Comment:
- extract zip archieve with chromedriver, which is installed manually (>73, <115)